tests/TestCase.php

```diff
@@ -155,6 +158,31 @@ protected function getTestName(bool $withDataSet = true): string
         return $this->getName($withDataSet);
     }
 
+    public static function assertHasIssueType(string $expected, string $message = ''): void
+    {
+        $issueMessages = [];
```

Comment:
Most of the codebase uses snake_case for variable/property names

Reply:
Thanks. I forgot about it. I'll fix it.